bip-0141.mediawiki

```diff
@@ -125,6 +125,20 @@ The sigop limit is likewise quadrupled to ≤ 80,000.
 
 Each P2WPKH input is counted as 1 sigop. In addition, opcodes within a P2WSH <code>witnessScript</code> are counted identically as previously within the P2SH <code>redeemScript</code>. That is, CHECKSIG is counted as only 1 sigop, and CHECKMULTISIG is counted as 1 to 20 sigops according to the arguments. This rule applies to both native witness program and P2SH witness program.
 
+=== Additional definitions ===
+
+The following definitions are used in IsStandardTx() policy and the RPC interface, but are not part of consensus limits.
```

Comment:
ok, I think I get the point about non-normative now. Strictly speaking, BIPs should be about the consensus protocol, so adding reference to policy and rfc commands doesn't really fit.

I'm happy with your language, but I'll explicitly say that the definitions aren't used for consensus limits.